akka-docs/rst/java/stream/stages-overview.rst

```diff
@@ -630,6 +641,17 @@ complete the current value is emitted downstream.
 
 **completes** when upstream completes
 
+reduce
+^^^^^^
+Start with first element and then apply the current and next value to the given function, when upstream
+complete the current value is emitted downstream. Similar to ``fold``.
+
+**emits** when upstream completes
```

Comment:
This isn't right, its more like `the result of the applied function when upstream emits an element`, no?